pymc/distributions/timeseries.py

```diff
@@ -104,19 +104,20 @@ def rng_fn(
 
         # If size is None then the returned series should be (1+steps,)
         if size is None:
-            init_size = 1
-            steps_size = steps
+            bcast_shape = np.broadcast_shapes(
```

Comment:
What does broadcast shape here do specifically that the code before was not doing?

Reply:
It makes sure that when `init` is larger than mu or sigma, the distribution has the right shape, i.e., `(*init.shape, nsteps)`